cdk/package.json

```diff
@@ -33,7 +33,7 @@
     "dynamo:destroy": "cdk destroy --app \"./mvnw -e -q compile exec:java -Dexec.mainClass=dev.stratospheric.todoapp.cdk.DynamoDbApp\" --force --require-approval never"
   },
   "devDependencies": {
-    "aws-cdk": "2.23.0"
+    "aws-cdk": "2.33.0"
```

Comment:
@BjoernKW FYI: the `main` branch was failing as we also always need to bump the CDK version here.